### DISCUSSION TITLE
Allow local optimizers?

### DISCUSSION THREAD

**Discussion by and3k5** [parcel-bundler/parcel#8659] [Ideas] [answered]
Would it be possible to allow local optimizers? Not for actual developing a public npm package but if I want to do something specific to my own code.

It would be nice to do something like this:
```JSON5
/* .parcelrc */
{
    "extends": "@parcel/config-default",
    "optimizers": {
        "*.js": [
            "...",
            "./license/parcel-optimizer-magic-stuff.js"
        ]
    }
}
```
Currently this results in the error **Parcel optimizer packages must be named according to "parcel-optimizer-{name}"** which I'm _not against_. But it doesn't make sense to validate a package name if it's a local path.

**Comment by devongovett** [comment, marked as answer]
https://parceljs.org/features/plugins/#local-plugins

#4321

**Reply by and3k5** [reply]
Argh, so sorry about this. Searched for local _optimizer_, not plugins.

Sorry about that 🤦‍♂️
